packages/nixpkgs-review/use_api_revs.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/nixpkgs_review/review.py b/nixpkgs_review/review.py
+index e8bb9ee..4552248 100644
+--- a/nixpkgs_review/review.py
++++ b/nixpkgs_review/review.py
+@@ -353,14 +353,14 @@ class Review:
+         if self.checkout == CheckoutOption.MERGE:
+             base_rev, pr_rev = fetch_refs(
+                 self.remote,
+-                pr["base"]["ref"],
+-                f"pull/{pr['number']}/merge",
++                pr["base"]["sha"],
++                pr["merge_commit_sha"],
+             )
+         else:
+             merge_rev, pr_rev = fetch_refs(
+                 self.remote,
+-                pr["base"]["ref"],
+-                f"pull/{pr['number']}/head",
++                pr["base"]["sha"],
++                pr["head"]["sha"],
```

Comment:
Same question.

Something like this may also fix the issue we see with merged PRs no longer being reviewable (because the `pull/<>/merge` ref is gone).